src/commands/hardis/org/diagnose/unusedusers.ts

```diff
@@ -179,43 +125,98 @@ export default class DiagnoseUnusedUsers extends SfdxCommand {
     }
 
     // Manage notifications
-    await this.manageNotifications(this.unusedUsers, summary);
+    await this.manageNotifications(this.unusedUsers);
+
+    if (this.unusedUsers.length > 0) {
+      uxLog(this, c.yellow(summary));
+      console.table(this.unusedUsers);
+    }
+    else {
+      uxLog(this, c.green(summary));
+    }
 
     // Return an object to be displayed with --json
     return {
       status: this.statusCode,
-      message: msg,
       summary: summary,
       unusedUsers: this.unusedUsers,
       csvLogFile: this.outputFile,
       xlsxLogFile: this.outputFilesRes.xlsxFile,
     };
   }
 
+  private async defineLicenseIdentifiers() {
+    if (!this.licenseIdentifiers) {
+      // Ask user if interactive mode
+      if (!this.licenseTypes && !isCI) {
+        const licenseTypesResponse = await prompts({
+          type: "select",
+          name: "licensetypes",
+          message: "Please select the period to detect inactive active users.",
+          choices: [
+            { title: 'all', value: 'all' },
+            { title: `all-crm`, value: 'SFDC,AUL,AUL1,AULL_IGHT' },
+            { title: `all-paying`, value: 'SFDC,AUL,AUL1,AULL_IGHT,PID_Customer_Community,PID_Customer_Community_Login,PID_Partner_Community,PID_Partner_Community_Login' },
+          ],
+        });
+        this.licenseTypes = licenseTypesResponse.licensetypes;
+      }
+      else {
+        this.licenseTypes = "all-crm";
+      }
+      // Get licenseIdentifiers from licenseType
+      this.licenseIdentifiers = this.licenseTypesCorrespondances[this.licenseTypes];
+    }
+  }
+
+  private async defineNumberOfInactiveDays() {
+    if (!this.lastNdays) {
+      if (!isCI) {
+        // If manual mode and days not sent as parameter, prompt user
+        const lastNdaysResponse = await prompts({
+          type: "select",
+          name: "days",
+          message: "Please select the period to detect inactive active users.",
+          choices: [
+            { title: `7 days`, value: 7 },
+            { title: `30 days`, value: 30 },
+            { title: `60 days`, value: 60 },
+            { title: `3 months (90 days)`, value: 90 },
+            { title: `6 months (180 days)`, value: 180 },
+            { title: `1 year (365 days)`, value: 365 },
+            { title: `2 years (730 days)`, value: 730 }
+          ],
+        });
+        this.lastNdays = lastNdaysResponse.days;
+      }
+      else {
+        this.lastNdays = 90;
+      }
+    }
+  }
+
   private async listUnusedUsersWithSfdcLicense(conn) {
-    let whereConstraint = `WHERE IsActive = true AND LastLoginDate != LAST_N_DAYS:${this.lastNdays} AND LastLoginDate != NULL`;
-    
+    let whereConstraint = `WHERE IsActive = true AND (` +
+      `(LastLoginDate != LAST_N_DAYS:${this.lastNdays} AND LastLoginDate != NULL) OR ` +
+      `(CreatedDate != LAST_N_DAYS:${this.lastNdays} AND LastLoginDate == NULL)` + // Check also for users never used
```

Comment:
Should this query be updated? 

FROM: AND LastLoginDate == NULL
TO: AND LastLoginDate = NULL